### DISCUSSION TITLE
Any demos or tips on how to enable collaborative editing?

### DISCUSSION THREAD

**Discussion by armaandh** [outline/rich-markdown-editor#198] [General] [answered]
Big thank you for the incredible project. Since you have transitioned to Prosemirror, are there any examples or pointers you could provide on how to enable a collaborative editing experience? Many thanks again for your efforts.

**Comment by tommoor** [comment, marked as answer]
Unfortunately not, I would refer to the Prosemirror documentation for that, I don't have any experience with it yet: https://prosemirror.net/examples/collab/

Prosemirror plugins can be passed directly to the editor through the `extensions` prop, I believe everything should be do-able through that interface but if not I'd entertain proposals for additions to the core editor to enable collab.

**Reply by armaandh** [reply]
sounds great. I'll try and get an example up and working for anyone interested.
